cafebabel/tests/test_articles.py

```diff
@@ -254,6 +355,30 @@ def test_delete_inexistent_article_should_return_404(client, editor, article):
     assert response.status_code == HTTPStatus.NOT_FOUND
 
 
+def test_editor_access_drafts_list(client, editor, article):
+    login(client, editor.email, 'secret')
+    response = client.get('/article/draft/')
+    assert response.status_code == HTTPStatus.OK
+    assert article.title in response.get_data(as_text=True)
+
+
+def test_author_cannot_access_drafts_list(client, user, article):
+    login(client, user.email, 'secret')
+    response = client.get('/article/draft/')
+    assert response.status_code == HTTPStatus.FORBIDDEN
+
+
+def test_drafts_list_only_displays_drafts(client, editor, article,
```

Comment:
And here.

Reply:
also moved to test_drafts and used for checking the presence of the title in the response